tests/integration/features/serverless_feature_test.go

```diff
@@ -28,6 +30,19 @@ const (
 	testDomainFooCom    = "*.foo.com"
 )
 
+const knativeServingSubscription = `apiVersion: operators.coreos.com/v1alpha1
```

Comment:
As we are adding more and more test fixture consts here maybe it's time to move them out to it's own file e.g. `test_fixtures/knative_resources.go`

I would rather jump straight to the test code and read what is going on instead of scrolling through a pile of "noise" at this point. WDYT?

Reply:
Agree, there is a ton of them (~50+ lines), doing this now.